pkgs/applications/networking/cluster/minikube/default.nix

```diff
@@ -14,23 +14,23 @@ let
   # instead, we download localkube ourselves and shove it into the minikube binary. The versions URL that minikube uses is
   # currently https://storage.googleapis.com/minikube/k8s_releases.json
 
-  localkube-version = "1.6.0";
+  localkube-version = "1.6.3";
```

Comment:
`https://storage.googleapis.com/minikube/k8sReleases/v${localkube-version}/localkube-linux-amd64` does not exist for 1.6.4